src/analysis/complex/isometry.lean

```diff
@@ -0,0 +1,185 @@
+/-
+Copyright (c) 2021 François Sunatori. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde, Yakov Pechersky, Heather Macbeth, Kevin Buzzard, Greg Price,
+François Sunatori
+-/
+import analysis.complex.basic
+import data.complex.exponential
+import data.real.sqrt
+import analysis.normed_space.linear_isometry
+
+/-!
+# Isometries of the Complex Plane
+
+Classification of isometries in the complex plane.
+
+## References
+
+* [Isometries of the Complex Plane](http://helmut.knaust.info/mediawiki/images/b/b5/Iso.pdf)
+-/
+
+-- https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/linear_isometry_complex
+-- https://github.com/leanprover-community/mathlib/pull/6923
+
+open complex
+
+local notation `|` x `|` := complex.abs x
+
+@[simp]
+lemma linear_isometry.id_apply (z : ℂ) : (linear_isometry.id : ℂ →ₗᵢ[ℝ] ℂ) z = z := rfl
+
+@[simp]
+lemma linear_isometry.id_to_linear_map : (linear_isometry.id.to_linear_map :
+  ℂ →ₗ[ℝ] ℂ) = linear_map.id := rfl
```

Comment:
It is not obvious to me that this belongs to this file, and also that this is stated in the right generality. For instance, wouldn't this be true over any complex vector space instead of just `ℂ`?